man/systemd.network.xml

```diff
@@ -1007,7 +1007,7 @@
           </listitem>
         </varlistentry>
         <varlistentry>
-          <term><varname>Protocol=</varname></term>
+          <term><varname>IPProtocol=</varname></term>
           <listitem>
             <para>Specifies the protocol to match in forwarding information base (FIB) rules. Accepted values are tcp, udp and sctp.
```

Comment:
while we are at it, I figure `tcp` and `udp` and `sctp` should be enclosed in `<literal></literal>` or so